src/marqo/core/models/hybrid_parameters.py

```diff
@@ -69,9 +69,11 @@ def validate_properties(cls, values):
 
         # score_modifiers_lexical can only be defined for Lexical, RRF, NormalizeLinear
         if values.get('scoreModifiersLexical') is not None:
-            if values.get('rankingMethod') not in [RankingMethod.Lexical, RankingMethod.RRF]:
+            if not (values.get('rankingMethod') in [RankingMethod.Lexical, RankingMethod.RRF] or
+                    values.get('retrievalMethod') == RetrievalMethod.Lexical):
                 raise ValueError(
-                    "'scoreModifiersLexical' can only be defined for 'lexical', 'rrf' ranking methods")  # TODO: re-add normalize_linear
+                    "'scoreModifiersLexical' can only be defined for 'lexical', 'rrf' ranking methods or "
+                    "'lexical' retrieval method.")  # TODO: re-add normalize_linear
 
         # score_modifiers_tensor can only be defined for Tensor, RRF, NormalizeLinear
         if values.get('scoreModifiersTensor') is not None:
```

Comment:
So just to confirm, the scoreModifiersTensor will not be applied to the first ranking phase of the tensor search, right?

Reply:
scoreModifiersTensor will be used in:
1. lexical/tensor search, in second phase.
2. tensor/lexical search, in first phase.

Reply:
So we'll need to add the same logic as `scoreModifiersLexical`, right?  I mean we should also allow this score modifier when `values.get('retrievalMethod') == RetrievalMethod.Tensor`

Reply:
No, since tensor/lexical is single phase (only lexical ranking)